nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -1415,14 +1415,14 @@ Utility modules can be either one of these types, just ensure you set its slot t
 /obj/item/fighter_component/secondary/ordnance_launcher/tier2
 	name = "Upgraded Fighter Missile Rack"
 	tier = 2
-	max_ammo = 5
+	max_ammo = 8
```

Comment:
Unrelated to the reported issue.

Reply:
Ah, forgot about that. The standard one currently holds 5 so the upgraded one is redundant. Meant to make them two separate PR's